lib/matplotlib/cbook.py

```diff
@@ -2234,6 +2235,28 @@ def _picklable_class_constructor(mixin_class, fmt, attr_name, base_class):
     return cls.__new__(cls)
 
 
+def _is_torch_array(x):
+    try:
+        return isinstance(x, sys.modules['torch'].Tensor)
+    except (KeyError, AttributeError, TypeError):
```

Comment:
I think a raw except would be safer here: for example, if someone defines a file named `torch.py` or `jax.py` that has a non-trivial module-level `__getattr__`, this could raise a `ValueError` or something else, and it would make `matplotlib` unusable with no real recourse. I think a raw`except` is preferable to that.

Reply:
Oh, just saw the warning below. I'm -1 on handing raw exceptions with a warning. We should just silently return `False` for all exceptions.

Reply:
@jakevdp I totally agree! For this case, I'd detach myself from "Errors should never pass silently." from "The Zen of Python, by Tim Peters" :)